### PR TITLE
docs: Ecosystem Page V2

### LINKED ISSUE #197: Ecosystem Page: New project form integration
Add a button directing to a Typeform allowing new project submissions

### PR DESCRIPTION
**Pull request by Rubyt0x**
New features:
- **Featured Projects:** a carousel featuring projects live on Testnet based on their categories.
- **Project Details side panel:** opens a side panel with more information about each project on a card click.
- **Informative Alert:** An alert tells people to click on cards.
- Added a `Add Project` button

Design enhancements:
- Card design enhancement: improved layout, social buttons

Fixes #197

### CODE REVIEW

**Review by luizstacio — CHANGES_REQUESTED** [review]
@Rubyt0x: 
1. can you add a cursor pointer when use hovers the card?
2. None of the buttons work... the only one that works is visit website with the project tab open. 
Screenshot 2024-01-10 at 10 44 36
Screenshot 2024-01-10 at 10 44 56
3. I think would be better to add the Visit Website button near to the logo, right now is easy to not see it.
Screenshot 2024-01-10 at 10 32 12

File: packages/app/src/systems/Ecosystem/components/FeaturedProjects/FeaturedProjects.tsx [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+              
+              .panelVisible {
+                animation: slideIn 0.5s forwards;
+              }
+              
+              .panelHidden {
+                animation: slideOut 0.5s forwards;
+              }
+        `}
+      </style>
```
Comment (luizstacio) [inline packages/app/src/systems/Ecosystem/components/FeaturedProjects/FeaturedProjects.tsx:162]: This should be in a css file.

**Review by luizstacio — CHANGES_REQUESTED** [review]
- Can you remove the inline styles, move them to the styles cost we have always creating a new item for it. Ex from `style={{ a: 1 }} to style={styles.myA}`
- You could also split more the code, and improve the use of state, but we can do this later.

**Review by LuizAsFight — CHANGES_REQUESTED** [review]
social buttons are grabbing too much attention, maybe try to do only bordered instead of background?

or keep same color? specially the X one when it's alone, that red button is the first thing you look

File: packages/app/src/systems/Ecosystem/components/FeaturedProjects/FeaturedProjects.tsx (hunk 1 above)
Comment (Rubyt0x) [reply packages/app/src/systems/Ecosystem/components/FeaturedProjects/FeaturedProjects.tsx]: Moved to animations.css

**Comment by LuizAsFight** [comment]
yes @Rubyt0x much better now, I will fix audit in other PR then you can get rid of CI error

**Review by LuizAsFight — APPROVED** [review]

**Review by LuizAsFight — CHANGES_REQUESTED** [review]
other changes requested from product side will come, so I'm blocking it for now

**Pull request closed without merging**